app/views/calendars/show.html.haml

```diff
@@ -0,0 +1,13 @@
+%p#notice= notice
+
+= month_calendar events: @events do |date, events|
+  = date
+
+  - events.each do |event|
+    = event.name
+    = event.location
```

Comment:
I added line 3 - 8. Info on how to do this can be found in the [readme](https://github.com/excid3/simple_calendar#rendering-events)